internal/scorecard/scorecard.go

```diff
@@ -224,13 +224,29 @@ func (r PodTestRunner) RunTest(ctx context.Context, test v1alpha3.TestConfigurat
 	// Create a Pod to run the test
 	podDef := getPodDefinition(r.configMapName, test, r)
 	if podSec {
-		secCtx := v1.PodSecurityContext{}
-		secCtx.RunAsNonRoot = &podSec
-		secCtx.SeccompProfile = &v1.SeccompProfile{
+		// creating a pod security context to support running in default namespace
+		podSecCtx := v1.PodSecurityContext{}
+		podSecCtx.RunAsNonRoot = &podSec
+		podSecCtx.RunAsUser = &[]int64{1000}[0]
+		podSecCtx.RunAsGroup = &[]int64{1000}[0]
```

Comment:
This is making sure that the container's don't run as root, and run as a specific user. User `0` is root, this is running as user `1000` which is perfectly acceptable. The issue is that PSA works differently in the `default` namespace then it does in other namespaces, and since scorecard runs in `default` all of this needs to be added in order to support that namespace.

Now if scorecard ran in some random namespace, these additions are less likely to be needed based on my testing. However, there are many tools that  use operator-sdk as a dependency, and I think rolling back the `default` namespace would be a breaking change for many users, as I mentioned in the issues.